docs/topics/backend.rst

```diff
@@ -48,19 +48,23 @@ On Mac the docker libs can be installed using Mac Homebrew::
 On Linux
 ~~~~~~~~
 
-See the `different instructions <https://docs.docker.com/installation/>`_ for your distribution.
+The generic (check docker link above for distribution specific) install script for docker is::
 
-The generic install curl-able script is::
+    wget -qO- https://get.docker.com/ | sh
```

Comment:
the docker installation pages changed to use wget rather than curl (I didn't look into the differences).  One is redundant now.  Any preference if we advise curl or wget?